model/task/task.go

```diff
@@ -1607,6 +1609,7 @@ func ResetTasks(taskIds []string) error {
 	return err
 }
 
+//TODO: (EVG-16305): create container version of this
```

Comment:
I don't think this comment is necessary since changing `ResetTasks` is already part of the ticket.